migrations/2_deploy_contract_set_for_tests.js

```diff
@@ -26,11 +26,45 @@ const PayoutHelper = artifacts.require('PayoutHelper')
 // Test mocks
 const OptionCallSyntheticIdMock = artifacts.require('OptionCallSyntheticIdMock')
 const DummySyntheticIdMock = artifacts.require('DummySyntheticIdMock')
+const OracleIdMock = artifacts.require('OracleIdMock')
 const TestToken = artifacts.require('TestToken')
 const WETH = artifacts.require('WETH')
 
 const baseTokenURI = 'https://explorer.opium.network/erc721o/'
 
+let registryInstanceAddress
+
+module.exports = async function(deployer, network, accounts) {
```

Comment:
I'd return this closure back and just pass the registry address to `deployMocks ` as an argument